src/platform/mt8195/include/platform/lib/memory.h

```diff
@@ -165,7 +165,11 @@
 #define HEAP_BUF_ALIGNMENT DCACHE_LINE_SIZE
 
 /** \brief EDF task's default stack size in bytes. */
+#ifdef CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING
+#define PLATFORM_TASK_DEFAULT_STACK_SIZE 8192
+#else
 #define PLATFORM_TASK_DEFAULT_STACK_SIZE 3072
+#endif
```

Comment:
It's actually the only place to put a stack size in RTOS.  FWIW the same kind of issue exists with the DP scheduler too, which shares a single worker thread for a core and thus has to be sized to the largest stack consumer that it might run, which is likewise done with a global setting, though there AEC was the first consumer so it got an 8k stack in the first version.